plugins/build/src/index.ts

```diff
@@ -275,12 +275,18 @@ export default class BuildPlugin implements Plugin<BuildArgs> {
             this.cssFiles[config.name] = new Map<string, postcss.Result>();
           }
 
+          const configPath = fs.existsSync(config.path)
+            ? config.path
+            : POSTCSS_CONFIG;
+
           const css = await transpileCSS({
             inFile: file,
             inDir: inputDirectory,
-            outDir: outputDirectory,
-            configFile: POSTCSS_CONFIG,
-            multiBuildConfigFile: config.path,
+            outDir:
+              config.name === this.buildArgs.cssMain
```

Comment:
Was there a particular reason you used the outDir instead of a new variable that's just `isMain` or something?

Reply:
I was trying to keep as much of the info about the multiBuild out of the postcss stuff. If we wanted to move all of that processing to the postcss module, I'd probably also opt to move all of the logic from [here](https://github.com/intuit/design-systems-cli/pull/514/files#diff-df27251910b01b549723eac3f6a709adL268) over that module as well so it's not scatted to much throughout